tools/makebadges.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+#
+# Generates markdown for README build status badges.
+#
+# Copyright 2025 James E. King III <jking@apache.org>
+#
+
+set -eu
+
+ARGS=$(getopt --longoptions "project:,appveyor:,codecov:,coverity:" "p:a:v:y:" -- "$@")
+
+if [ $? != 0 ] ; then echo "Terminating..." >&2 ; exit 1 ; fi
+
+eval set -- "$ARGS"
+
+PROJECT=
+APPVEYOR=
+CODECOV=
+COVERITY=
+
+while true; do
+  case "${1:-*}" in
+    -p | --project) PROJECT="$2"; shift 2 ;;
+    -a | --appveyor) APPVEYOR="$2"; shift 2 ;;
```

Comment:
This doesn't seem to be enough. For Boost.CI:
`[![Build status](https://ci.appveyor.com/api/projects/status/ynnd2l3gu4oiyium/branch/master?svg=true)](https://ci.appveyor.com/project/Flamefire/boost-ci/branch/master)`

So we need both the appveyor project owner and that hash/id-thing